--- __init__ (copie).py	2011-02-02 19:20:28.000000000 +0100
+++ __init__.py	2011-02-02 19:21:17.021348002 +0100
@@ -5,6 +5,8 @@
 from django.conf import settings
 import sys
 
+ignore_dbs = getattr(settings, 'SOUTH_IGNORE_DATABASES', [])
+
 # A few aliases, because there's FQMNs now
 engine_modules = {
     'django.db.backends.postgresql_psycopg2': 'postgresql_psycopg2',
@@ -36,7 +38,7 @@
     db_engines = dict([
         # Note we check to see if contrib.gis has overridden us.
         (alias, "south.db.%s" % engine_modules.get(db_settings['ENGINE'], None))
-        for alias, db_settings in settings.DATABASES.items()
+        for alias, db_settings in settings.DATABASES.items() if alias not in ignore_dbs
     ])
     # Update with any overrides
     db_engines.update(getattr(settings, "SOUTH_DATABASE_ADAPTERS", {}))
